cardshifter-core/src/main/java/com/cardshifter/core/groovy/EffectDescription.groovy

```diff
@@ -0,0 +1,55 @@
+package com.cardshifter.core.groovy
+
+class EffectDescription {
+
+    /**
+     * The description (value of map) should be cased and punctuated as if in the middle of a sentence. This makes it
+     * possible to later change the ordering of description components. It's easier to capitalize and add punctuation
+     * later than the reverse. A map is useful instead of enum et al because it's runtime extensible. */
+    static Map<String, String> vocabulary = new HashMap()
+
+    // Key into vocabulary
+    String triggerId
+
+    // choose $randomChoiceCount atRandom { $randomChoices... }
+    int randomChoiceCount
+    List<EffectDescription> randomChoices
+
+    private StringBuilder builder = new StringBuilder()
+
+    public static setupStandardVocabulary() {
```

Comment:
should perhaps be a "static initialisation block", or are we using this because different mods might want to do it differently or something? Also: Please specify return value.